app/src/main/java/fr/neamar/kiss/DataHandler.java

```diff
@@ -72,6 +77,11 @@ public DataHandler(Context context) {
         if (prefs.getBoolean("enable-alias", true)) {
             providers.add(new AliasProvider(context, appProvider));
         }
+        shortcutProvider = (new ShortcutProvider(context));
```

Comment:
maybe add a comment explaining why we're creating a shortcut provide even if the settings is unchecked?

// We need to create the shortcutrprovider in every case, since we can get a new INSTALL_SHORTCUT intent at any time